ui/src/components/SingleInputComponent/SingleInputComponent.tsx

```diff
@@ -155,16 +155,16 @@ function SingleInputComponent(props: SingleInputComponentProps) {
                 }
             })
             .catch(() => {
-                if (current) {
+                if (mounted) {
                     setLoading(false);
+                    setOptions([]);
                 }
-                setOptions([]);
             });
 
         // eslint-disable-next-line consistent-return
         return () => {
-            abortController.abort('Operation canceled.');
-            current = false;
+            mounted = false;
```

Comment:
the order is changed to make sure the `mounted` has false value in `catch` block